scripts/zkai_d128_recursive_pcd_route_selector_gate.py

```
@@ -0,0 +1,1208 @@
+
+import argparse
+import copy
+import csv
+import hashlib
+import io
+import json
+import pathlib
+import sys
+import tempfile
+import tomllib
+from typing import Any, Callable
```

Ruff (0.15.12)

[warning] Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)